src/Planners/src/UnicycleTrajectoryGenerator.cpp

```diff
@@ -276,6 +283,14 @@ bool Planners::UnicycleTrajectoryGenerator::initialize(
                                                           initialBasePosition,
                                                           leftToRightTransform);
 
+    // Get the generator being used and instantiate optional outputs
+    ok = ok && loadParamWithFallback("use_zmp_generator", m_useZMPGenerator, false);
+    m_useDCMGenerator = !m_useZMPGenerator;
```

Comment:
I would still give the possibility of using both of them at the same time

Reply:
Done with [4e1a9a5](https://github.com/ami-iit/bipedal-locomotion-framework/pull/916/commits/4e1a9a5a14c2fb267a3b37847c5cc92ca25276ce)